tests/unit/contextMenu.spec.js

```diff
@@ -377,4 +377,168 @@ describe("ContextualMenu", () => {
       });
     });
   });
+
+  describe("Opening search result options", () => {
+    it("Should focus new tab depending on resultsInBackgroundTab flag", async () => {
+      const info = {
+        menuItemId: "searchprovider-0",
+        selectionText: "resultsInBackgroundTab",
+      };
+      const tab = {
+        index: 10,
+      };
+      const settings = await LocalStore.getOne(StoreKey.SETTINGS);
+      const providers = await LocalStore.getOne(StoreKey.SEARCH_PROVIDERS);
+      const provider = _.find(providers, function (item) {
+        return item.menuIndex === info.menuItemId;
+      });
+      const targetURL = getProviderTargetURL(provider, info.selectionText);
+
+      // If shouldn't focus on new tab.
+      await LocalStore.setOne(StoreKey.SETTINGS, {
+        ...settings,
+        resultsInBackgroundTab: true,
+      });
+      await onClickedListener(info, tab);
+      expect(createTabs).toBeCalledWith({
+        url: targetURL,
+        selected: false,
+        index: tab.index + 1,
+      });
+
+      // If should focus on new tab.
+      await LocalStore.setOne(StoreKey.SETTINGS, {
+        ...settings,
+        resultsInBackgroundTab: false,
+      });
+      await onClickedListener(info, tab);
+      expect(createTabs).toBeCalledWith({
+        url: targetURL,
+        selected: true,
+        index: tab.index + 1,
+      });
```

Comment:
This should be split into two unit tests: tab is focused if flag is true, and tab isn't focused if tab is false